docs: add RBAC docs (#2106)
emqtt/emq-docs-cn: en_US/dashboard/system.md

Click **System** -> **Users** to enter the **Users** page. The user page displays a list of all current users who can log in to Dashboard, including users created using [CLI](../admin/cli.md). Click on the **+ Create** button in the top right corner of the page to open the Create User pop-up box, fill in the user information and click on the **Create** button to submit the data. Users that are successfully submitted will be added to the list of users. Users that are successfully added can have their passwords changed or their notes changed on the page; users that are no longer required can be deleted on the page.

{% emqxce %}
<img src="./assets/users.png" alt="image" style="zoom:67%;" />

> All users have admin rights to delete other users, but you cannot delete the currently logged in user on the Dashboard.
> For security reasons, starting with EMQX 5.0.0, Dashboard user cannot be used for REST API authentication.
{% endemqxce %}

{% emqxee %}
<img src="./assets/ee-users.png" alt="image" style="zoom:67%;" />

> Since EMQX 5.3 we introduced the RBAC (Role-based access control) feature for the Dashboard users.

Reviewer: ```suggestion
Since EMQX 5.3 we introduced the RBAC (Role-based access control) feature for the Dashboard users.
RBAC allows us to assign permissions to users based on their roles in the organization.RBAC enables simplified authorization management, enhances security through restricted access, and improves organizational compliance, making it an essential access control mechanism for the Dashboard.
```

Currently, there are two predefined roles:

> + administrator

Reviewer: ```suggestion
+ Administrator
```

>     This role could access all resources.

Reviewer: ```suggestion
    Has full access to manage all features and resources of EMQX, including client management, system configuration, API key and user management.
```

> + viewer

Reviewer: ```suggestion
+ Viewer
```

>     This role can only view resources and data, corresponding to all `GET` requests in the REST API.

Reviewer: ```suggestion
    Can access all data and configuration of EMQX, corresponding to all `GET` requests in the REST API. But has no rights to create, modify or delete.
```
